communication/OWNERS

```diff
@@ -2,16 +2,17 @@
 
 reviewers:
   - parispittman
-  - castrojo
   - jdumars
   - idvoretskyi
   - jeefy
   - mrbobbytables
   - markyjackson-taulia
 approvers:
-  - castrojo
+  - mrbobbytables
   - parispittman
   - sig-contributor-experience-leads
+emeritus_approvers:
+  - castrojo
```

Comment:
@castrojo , Would you like to remain as an emeritus approver in the communication owners file?